plugin/src/main/java/org/screamingsandals/bedwars/game/RespawnProtection.java

```diff
@@ -23,11 +23,14 @@ public RespawnProtection(Game game, Player player, int seconds) {
     public void run() {
     	if (!running) return;
         if (length > 0) {
-            MiscUtils.sendActionBarMessage(player, i18nonly("respawn_protection_remaining").replace("%time%", String.valueOf(this.length)));
-
+            if (Main.getConfigurator().config.getBoolean("respawn.show-messages")) {
+                MiscUtils.sendActionBarMessage(player, i18nonly("respawn_protection_remaining").replace("%time%", String.valueOf(this.length)));
```

Comment:
i'd prefer using `Integer.toString(length)` over `String.valueOf(this.length)` to keep code consistency

Reply:
ok